core/src/main/java/org/infinispan/interceptors/distribution/DistributionBulkInterceptor.java

```diff
@@ -378,4 +381,20 @@ public TxBackingKeySet(Cache<K, V> cache, CacheSet<CacheEntry<K, V>> entrySet, F
                  StreamMarshalling.entryToKeyFunction(), ctx);
       }
    }
+
+   @Override
+   public Object visitGetKeyValueCommand(InvocationContext ctx, GetKeyValueCommand command)
+         throws Throwable {
+      return invokeNext(ctx, command);
```

Comment:
Might want to leave a comment why this is here so someone doesn't remove it by accident later :)